### PR TITLE
Fix wording in `LookAtModifier3D` docs

### LINKED ISSUE #99532: LookAtModifier3D documentation data has typos
### Tested versions

- Reproducible in: Godot v4.4.dev5 snapshot

### System information

Godot v4.4.dev5 - Pop!_OS 22.04 LTS on X11 - X11 display driver, Multi-window, 1 monitor - Vulkan (Forward+) - dedicated AMD Radeon RX 6750 XT (RADV NAVI22) - AMD Ryzen 5 5600 6-Core Processor (12 threads)

### Issue description

After the introduction of [Implement LookAtModifier3D](https://github.com/godotengine/godot/pull/98446), the newly created documentation data (`doc/classes/LookAtModifier3D.xml`) for the LookAtModifier3D has some typos where it uses the name `SkeletonModifier3D` instead of the correct `LookAtModifier3D`

This is visible at the documentation page at [https://docs.godotengine.org/en/latest/classes/class_lookatmodifier3d.html](https://docs.godotengine.org/en/latest/classes/class_lookatmodifier3d.html)

Location 1:
`The SkeletonModifier3D rotates a bone to look a target.`
it should say
`The LookAtModifier3D rotates a bone to look a target.`

Location 2:
`This SkeletonModifier3D rotates a bone to look a target. [...]`
it should say
`This LookAtModifier3D rotates a bone to look a target. [...]`

And on `Create a new node` window on the tooltip and on the description section
image

### Steps to reproduce

Documentation page:
1. Go to https://docs.godotengine.org/en/latest/classes/class_lookatmodifier3d.html

Inside the Godot's editor:
1. Open the `Create a new node` window
2. Select the `LookAtModifier3D` node

### Minimal reproduction project (MRP)

-

### PR DESCRIPTION
**Pull request by crei0**
Fixed some typos and wording in documentation

* *Bugsquad edit, fixes: https://github.com/godotengine/godot/issues/99532*

### CODE REVIEW

**Review by AThousandShips — DISMISSED** [review]

File: doc/classes/LookAtModifier3D.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LookAtModifier3D" inherits="SkeletonModifier3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		The [SkeletonModifier3D] rotates a bone to look a target.
+		The [LookAtModifier3D] rotates a bone to look a target.
 	</brief_description>
 	<description>
-		This [SkeletonModifier3D] rotates a bone to look a target. This is helpful for moving character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
+		This [LookAtModifier3D] rotates a bone to look a target. This is helpful for moving character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
```
[the remark this answers is not in the document]
Comment (crei0) [reply doc/classes/LookAtModifier3D.xml]: Reverted the change

[the remark this answers is not in the document]
Comment (AThousandShips) [reply doc/classes/LookAtModifier3D.xml]: Also to fix an actual typo here:
```suggestion
		This [SkeletonModifier3D] rotates a bone to look at a target. This is helpful for moving a character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
```

File: doc/classes/LookAtModifier3D.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LookAtModifier3D" inherits="SkeletonModifier3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		The [SkeletonModifier3D] rotates a bone to look a target.
+		The [LookAtModifier3D] rotates a bone to look a target.
```
Comment (AThousandShips) [inline doc/classes/LookAtModifier3D.xml:4]:
```suggestion
		The [LookAtModifier3D] rotates a bone to look at a target.
```
Same here

File: doc/classes/LookAtModifier3D.xml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="LookAtModifier3D" inherits="SkeletonModifier3D" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		The [SkeletonModifier3D] rotates a bone to look a target.
+		The [LookAtModifier3D] rotates a bone to look at a target.
 	</brief_description>
 	<description>
-		This [SkeletonModifier3D] rotates a bone to look a target. This is helpful for moving character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
+		This [SkeletonModifier3D] rotates a bone to look at a target. This is helpful for moving character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
```
Comment (AThousandShips) [inline doc/classes/LookAtModifier3D.xml:7]:
```suggestion
		This [SkeletonModifier3D] rotates a bone to look at a target. This is helpful for moving a character's head to look at the player, rotating a turret to look at a target, or any other case where you want to make a bone rotate towards something quickly and easily.
```

File: doc/classes/LookAtModifier3D.xml (hunk 2 above)
Comment (crei0) [reply doc/classes/LookAtModifier3D.xml]: Improved the wording there and on line 7, thank you

File: doc/classes/LookAtModifier3D.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (crei0) [reply doc/classes/LookAtModifier3D.xml]: Added the missing `a` after `moving`

**Review by Mickeon — APPROVED** [review]

**Comment by akien-mga** [comment]
Looks good! Before merging the commits need to be squashed to match our [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) (see that page for instructions).

As it's your first contribution, let us know if this seems a bit daunting, we're also happy to do it for you to ease the process.

**Comment by crei0** [comment]
@akien-mga , I've squashed the commits

**Comment by AThousandShips** [comment]
Please update your commit message to be more descriptive, the title of the PR would be appropriate

**Comment by crei0** [comment]
@AThousandShips done, thank you

**Review by akien-mga — APPROVED** [review]

**Review by fire — APPROVED** [review]

**Review by TokageItLab — APPROVED** [review]
Thank you for the following up!

**Comment by Repiteo** [comment]
Thanks! Congratulations on your first merged contribution! 🎉

**Pull request merged**